### DISCUSSION TITLE
How to guide for 1.9 inch display.

### DISCUSSION THREAD

**Discussion by dev-techshlok** [Bodmer/TFT_eSPI#2370] [Q&A - General]
Hello, 
I have been using this library for a while on 1.47-inch display,
Any idea on how I can make this 1.9-inch display work with this library?
https://www.waveshare.com/product/displays/lcd-oled/lcd-oled-3/1.9inch-lcd-module.htm

Thank you!

**Comment by Bodmer** [comment]
[Setup 72](https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup72_ESP32_ST7789_172x320.h) should work with an ESP32.
